<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="Alshawabkeh" name="test.CareNetSettings.DataManagement.BodySystem" tests="7" failures="2" timestamp="2020-02-12T10:41:53 EET" time="145.627" errors="0">
  <testcase name="navigateToBodySystem" time="24.530" classname="test.CareNetSettings.DataManagement.BodySystem"/>
  <testcase name="addBodySystem" time="30.726" classname="test.CareNetSettings.DataManagement.BodySystem"/>
  <testcase name="editBodySystem" time="29.773" classname="test.CareNetSettings.DataManagement.BodySystem"/>
  <testcase name="addBodySystemPartList" time="30.196" classname="test.CareNetSettings.DataManagement.BodySystem">
    <failure type="java.lang.AssertionError" message="Operation Done Successfully . expected [x
Operation Done Successsfully .] but found [x
Operation Done Successfully .]">
      <![CDATA[java.lang.AssertionError: Operation Done Successfully . expected [x
Operation Done Successsfully .] but found [x
Operation Done Successfully .]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at Driver.BasePage.AssertOperationDoneSuccessfully(BasePage.java:418)
at test.CareNetSettings.DataManagement.BodySystem.addBodySystemPartList(BodySystem.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- addBodySystemPartList -->
  <testcase name="editBodySystemPartList" time="13.154" classname="test.CareNetSettings.DataManagement.BodySystem">
    <skipped/>
  </testcase> <!-- editBodySystemPartList -->
  <testcase name="deleteBodySystemPartList" time="0.000" classname="test.CareNetSettings.DataManagement.BodySystem">
    <skipped/>
  </testcase> <!-- deleteBodySystemPartList -->
  <testcase name="deleteBodySystem" time="17.248" classname="test.CareNetSettings.DataManagement.BodySystem">
    <failure type="java.lang.AssertionError" message="Operation Done Successfully . expected [x
Operation Done Successfully .] but found [x
Can&#039;t Be Deleted Cause This Body System Has Related Parts]">
      <![CDATA[java.lang.AssertionError: Operation Done Successfully . expected [x
Operation Done Successfully .] but found [x
Can't Be Deleted Cause This Body System Has Related Parts]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at test.CareNetSettings.DataManagement.BodySystem.deleteBodySystem(BodySystem.java:135)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- deleteBodySystem -->
</testsuite> <!-- test.CareNetSettings.DataManagement.BodySystem -->
